main.bundle.js

```diff
@@ -44,8 +44,53 @@
 /* 0 */
 /***/ (function(module, exports) {
 
+	'use strict';
+
 	// This file is in the entry point in your webpack config.
-	"use strict";
+
+	// 1. Weather for a location:
+	// - GET /api/v1/forecast?location=denver,co
+	// 2. Account Creation:
+	// - POST /api/v1/users
+	// 3. Login/Logout
+	// - POST /api/v1/sessions
+	// 4. Favoriting Locations
+	//  - POST /api/v1/favorites
+	// 5. Listing Favorite Locations
+	//  - GET /api/v1/favorites?api_key={'blahblah'}
+	// 6. Removing Favorite Locations
+	//  - DELETE /api/v1/favorites
+
+	var productionUrl = 'https://rain-or-shine-1.herokuapp.com';
+
+	function getWeather(location) {
+	  debugger;
+	  var location = location;
+	  var url = productionUrl + '/api/v1/forecast?location=' + location;
+	  fetch(url).then(function (response) {
+	    return response.json();
+	  }).then(function (weatherResponse) {
+	    return displayWeatherDetails(weatherResponse).catch(function (error) {
```

Comment:
In your index you’re calling this.displayWeatherDetails. But  not here.  Did you add the code to main bundle or is that automatic? I’m rusty on the procedure for this